website/www/site/content/en/documentation/programming-guide.md

```diff
@@ -499,16 +531,34 @@ the transform itself as an argument, and the operation returns the output
 [Output PCollection] = [Input PCollection] | [Transform]
 {{< /highlight >}}
 
+{{< highlight go >}}
+[Output PCollection] := beam.ParDo(s, [Transform], [Input PCollection])
+{{< /highlight >}}
+
+{{< paragraph class="language-java language-py" >}}
 Because Beam uses a generic `apply` method for `PCollection`, you can both chain
 transforms sequentially and also apply transforms that contain other transforms
 nested within (called [composite transforms](#composite-transforms) in the Beam
 SDKs).
+{{< /paragraph >}}
+
+{{< paragraph class="language-go" >}}
+Because Go doesn't support function overloading, it's recommended to
```

Comment:
We can probably remove `"Because Go doesn't support function overloading..."` and leave just the recommendation. Users don't really need to understand the technical details of why took this approach, just what to do.

Reply:
Agreed. Done.